docs/wallets.md

```diff
@@ -0,0 +1,57 @@
+# For Wallet Developers
+
+codex32 is a new format for BIP32 master seeds. It is essentially a replacement for
+BIP39 or SLIP39 seed words, and the user workflow should be much the same, except
+that:
+
+* The character set is different (bech32 characters rather than a wordlist).
+* Seeds may be encoded across multiple shares, rather than a single string.
+* It is possible to do specific error detection/correction during entry.
+
+There are two levels of wallet support:
+
+* The ability to import seeds/shares; here we essentially just have recommendations about dealing with errors.
+* The ability to generate seeds/shares on the device; here our guidance is more involved.
+
+We encourage every wallet to support importing seeds and shares, since
+* the technial difficulty is low (roughly on par with that of supporting Segwit addresses, plus optional error-correction support);
+* the added functionality is isolated from the rest of the wallet (once the seed is imported you don't care where it came from); and
+* beyond correctness of the code, there is little risk (no need to source randomness or execute potentially variable-time algoritms).
+
+Supporting seed generation is a little more involved so the tradeoff between
+implementation complexity and user value is less clear, especially since the
+Codex provides users instructions on doing generation themselves.
+
+## Import Support
+
+codex32 shares may be any length between 128 and 512 bits.
+Wallets should support import of 128- or 256-bit seeds; other lengths are optional.
+
+128-bit seeds are 48 characters in length, including the `MS1` prefix.
+256-bit seeds are 74. For other bitlengths, see the BIP.
+
+The process for entering shares is:
+
+1. The user should select the bit length of the import before entering any actual data.
+1. Then the user should enter the first share. To the entext possible given screen limitations, when entering share data, data should be displayed in uppercase, visually separated into four-character windows. The first four-character window should include the `MS1` prefix, which should be pre-filled.
+1. Once the first share is fully entered, the wallet should validate the checksum and header before accepting it.
+   * The user should not be able to entire mixed-case or non-bech32 characters.
```

Comment:
Users definitely need to be able to enter at least a `?` and in theory accepting non-bech32 characters could be helpful for improving desperate error recovery. (e.g. converting O to 0s).

Reply:
Typing a non-bech32 character that has a common confusion B to 8, I to l should try the common substitutions first then turn it into an erasure by substituting ?, other non-bech can be immediately marked as ?, perhaps even as the user types to educate them of  the CHARSET.

If the string has more than 8 ? or 13 contiguous (or the combos of contiguous and random mentioned in my error correction thread) the "Submit" button could change to "Attempt Repair" or "Attempt Correction" which will ask if the share set was generated by electronics. Standardized implementations in #57 will have some additional error detection bandwidth in the identifier, payload padding and potentially even share payloads themselves (tbd). Handmade or non-compliant electronic implementations will need to provide a known address to "overclock" the erasure correction.

Hypothetically, this option could appear if more than 4 substitution errors are present as well.